config/templates/cloud-config-etcd

```diff
@@ -86,6 +87,17 @@ coreos:
 
 
 write_files:
+
+  - path: /etc/environment
+    permissions: 0644
+    content: |
+      COREOS_PUBLIC_IPV4=$public_ipv4
+      COREOS_PRIVATE_IPV4=$private_ipv4
+      ETCDCTL_CA_FILE=/etc/etcd2/ssl/ca.pem
+      ETCDCTL_CERT_FILE=/etc/etcd2/ssl/etcd-client.pem
+      ETCDCTL_KEY_FILE=/etc/etcd2/ssl/etcd-client-key.pem
+      ETCDCTL_ENDPOINT=
```

Comment:
I realized that this line is the "marker" for https://github.com/coreos/kube-aws/pull/58/files#diff-c88cb625705bf9711a2f576a1975cb72R34.
Doing this makes it possible to:
* Mention that we have to provided an actual value when running `etcdctl`
* Ensure two things:
  * Appropriate `ETCDCTL_ENDPOINT` value is always configured on starting/restarting `etcd2.service`, automatically.
  * Restarting `etcd2.service` doesn't break `/etc/environment` as we can see in https://github.com/coreos/kube-aws/pull/58/files#diff-c88cb625705bf9711a2f576a1975cb72R34
    * For example, `echo ETCDCTL_ENDPOINT="https://$(hostname):2379/" >> /etc/environment` would break, or needlessly bloat the environment file at least, when `etcd2.service` got restarted.